qiskit_ibm_provider/job/ibm_circuit_job.py

```diff
@@ -938,13 +965,13 @@ def _status_callback(
             callback(self.job_id(), status, self, queue_info=queue_info)
 
     def _get_status_position(
-        self, api_status: str, api_info_queue: Optional[Dict] = None
+        self, api_status: str, api_metadata: Optional[Dict] = None
```

Comment:
Looking at the callers of this method, I don't think any of them actually pass the correct `api_metadata`, which is from the `metrics` endpoint

Reply:
we can pass in the `api_metadata`, but compared to the `QueueInfo` class, the runtime metadata doesn't contain any queue information 

```
{'timestamps': {'created': '2022-09-22T02:11:13.383086Z', 'finished': '2022-09-22T02:24:40Z', 'running': '2022-09-22T02:24:32Z'}, 'bss': {'seconds': 4}, 'executions': 10, 'num_circuits': 1, 'num_qubits': [5], 'circuit_depths': [5], 'qiskit_version': '0.7.0rc2,0.21.2,0.11.0', 'estimated_start_time': '2022-09-22T02:24:28.416Z', 'estimated_completion_time': '2022-09-22T02:24:32.416Z'}
```

Reply:
`estimated_start_time` and  `estimated_completion_time` are both attributes of `QueueInfo`. It's supposed to return `position_in_queue` and priority as well if the job is in queued state.